packages/next/components/listInfo/details.js

```diff
@@ -90,7 +90,10 @@ export default function Details({ space }) {
         <DetailsItem>
           <DetailsLabel>Threshold</DetailsLabel>
           <DetailsValue>
-            {toPrecision(space.proposeThreshold, space.decimals)} {space.symbol}
+            {abbreviateBigNumber(
```

Comment:
can we apply a <ValueDisplay> here?

Reply:
Not sure for now, the issue requires `(remove tooltip effect)`, I am asking the designer for details.